pkgs/development/python-modules/find-libpython/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "find_libpython";
```

```suggestion
  pname = "find-libpython";
```

Should be [normalized](https://nixos.org/manual/nixpkgs/stable/#contributing).